wcomponents-theme/src/test/intern/wc.file.util.test.js

```diff
@@ -0,0 +1,71 @@
+define(["intern!object", "intern/chai!assert", "./resources/test.utils!"],
+	function (registerSuite, assert, testutils) {
+		"use strict";
+		var fileUtil,
+			wcconfig;
+		registerSuite({
+			name: "wc/file/util",
+			setup: function () {
+				return testutils.setupHelper(["wc/file/util", "wc/config"], function (util, config) {
+					fileUtil = util;
+					wcconfig = config;
+				});
+			},
+			testBlobToFile: function () {
+				var expectedMime = "text/plain",
+					expectedFile = "test.file",
+					blob = new Blob(["This is my blob content"], {type: expectedMime}),
+					file = fileUtil.blobToFile(blob, {
+						name: expectedFile
+					});
+				assert.strictEqual(file.name, expectedFile);
+				assert.strictEqual(file.type, expectedMime);
+			},
+			testDataURItoBlob: function () {
+				var expectedText = "hello world",
+					expectedMime = "text/plain",
+					dataurl = 'data:'+ expectedMime +';base64,' + btoa(expectedText),
+					fileReader = new FileReader(),
+					blob;
+
+				blob = fileUtil.dataURItoBlob(dataurl);
+				new Promise(function (win) {
+					fileReader.onload = function () {
+						assert.strictEqual(fileReader.result, expectedText);
```

Comment:
Just for your information (you don't need to change this) I would tend to write my event handlers in a more portable and reusable way (not relying on the scope), i.e.
```javascript
fileReader.onload = function (event) {
    assert.strictEqual(event.target.result, expectedText);
```

In this particular case it really doesn't matter because you can't easily move the event handler AND it's very small